### PR TITLE
Installer fixes, reflecting the new pom file structure #8494

### LINKED ISSUE #8494: Vagrant Deployment not working: 'can't seem to find an appropriate warfile.'
**What steps does it take to reproduce the issue?**

* When does this issue occur?
When trying to deploy dataverse using vagrant

* Which page(s) does it occurs on?
N/A

* What happens?
Running `vagrant up` starts the deployment process, but after configuring Solr the process fails because the python installer is unable to find a .war file:
```
==> default: Running provisioner: shell...
    default: Running: /tmp/vagrant-shell20220316-16786-1agei0k.sh
    default: enabling non-interactive mode
    default: 
    default: This appears to be a RedHat system; good.
    default: Checking for required components...
    default: Sorry, I can't seem to find an appropriate warfile.
    default: Are you running the installer from the right directory?
```

* To whom does it occur (all users, curators, superusers)?

* What did you expect to happen?
Deployment using `vagrant up` should be successful

**Which version of Dataverse are you using?**
5.9

**Any related open or closed issues to this bug report?**
Not that I could find

**Screenshots:**

No matter the issue, screenshots are always welcome.

To add a screenshot, please use one of the following formats and/or methods described here:

* https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests
*

### PR DESCRIPTION
**Pull request by j-n-c**
**What this PR does / why we need it**:
Fix python installer search for dataverse .war file. Currently deploying dataverse with vagrant is not working.
[from @landreev:]  The PR now includes the fixes for the old school (perl) installer, and the installer Makefile as well. 

**Which issue(s) this PR closes**:

Closes #8494

**Special notes for your reviewer**:
Check if the revision field in modules/dataverse-parent/pom.xml is a suitable location for retrieving dataverse version

**Suggestions on how to test this**:

[from @landreev:] It should now be possible to make the installer bundle; and run the installer successfully from the source tree. 
In other words, the QA for this PR is one of the standard steps of making a release. This means this issue would be found eventually - but this PR removes the element of surprise from the process. 

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

**Is there a release notes update needed for this change?**:

**Additional documentation**:

### CODE REVIEW

**Comment by landreev** [comment]
I'd be happy to add the perl installer fix to this PR.

**Comment by landreev** [comment]
Of course the makefile that builds the installer bundle is also broken, for the same very reason. I can add a fix for that too.
